### ISSUE TITLE
Debounce on authentication settings is too short when typing into an input

### ISSUE THREAD

**Issue by AnthonyMde** [hopfront/hopfront#120]

**Comment by remihrt** [comment]
What should be tested?

**Comment by AnthonyMde** [comment]
You should go to settings > authentification > choose a static authentification mode > try to type the key name.
It was 500ms: if the user was a bit slow, conf saving was stopping him in the middle of its typing. 
I set it to 1000ms: check if balance between quick saving / comfort is OK.
FYI, you may encounter this bug during your test : https://github.com/hopfront/hopfront/issues/121

**Comment by remihrt** [comment]
Okay thank you.

I tested it and I think the problem is that the input is losing focus when saving is done.
Even with 1000ms it could be annoying.

If the input wouldn't lose focus the saving could be trigger even with less than 500ms of inactivity.

I would expect that the saving would be trigger if the input lose focus or if enter is press.

**Comment by AnthonyMde** [comment]
Yes, it was a work around as I saw it during development but it was not prioritize. 
The saving method is linked to other auth components, so it's not a quick task. 
Should I rework on it or create a new ticket and you will set its priority?
